.husky/pre-push

```diff
@@ -0,0 +1,4 @@
+cd frontend
+npx lint-staged
+cd ../backend
+ruff check
```

Comment:
This has a wider impact on the backend team, I would add this to the retro board to discuss with the team as a whole. Happy for frontend pre-push/pre-commit hooks to be agreed between you, me and Mike

Reply:
So, for now, I would remove line 3-4 and keep 1-2 :)